cluster/gce/gci/configure-helper.sh

```diff
@@ -1127,6 +1139,9 @@ function start-kube-apiserver {
   params+=" --secure-port=443"
   params+=" --tls-cert-file=${APISERVER_SERVER_CERT_PATH}"
   params+=" --tls-private-key-file=${APISERVER_SERVER_KEY_PATH}"
+  params+=" --requestheader-client-ca-file=${AGGREGATOR_CA_CERT_PATH}"
```

Comment:
talked on slack... I think this is the actual issue... this is only part of the requestheader config.

You should also indicate what the expected CN of the aggregator's proxy client cert is, and the headers to use to convey username/group/extra info, something like this:
```
--requestheader-allowed-names=aggregator
--requestheader-extra-headers-prefix=X-Remote-Extra-
--requestheader-group-headers=X-Remote-Group
--requestheader-username-headers=X-Remote-User
```